targets/unity-v2/source/ExampleTestProject/Assets/PlayFabSDK/Shared/Public/PlayFabEvents.cs.ejs

```diff
@@ -52,30 +65,75 @@ for (var i in api.calls) { var apiCall = api.calls[i]; var prefix = ""; if (api.
             }
         }
 
+<% for (var x = 0; x < apis.length; x++) {
```

Comment:
just more comments here.  the formatting is somewhat better, i find. should be improved throughout.